index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Startup Matchmaker</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+  </head>
+
+  <body>
+    <div class="container">
+
+      <header>
+        <h1>Startup Matchmaker</h1>
+        <nav>
+          <a href="#" class="caps">Find Developers</a>
+          <a href="#" class="caps">Find Designers</a>
+          <a href="#">How it Works</a>
+          <a href="#">Our Team</a>
+          <a href="#">Blog</a>
+        </nav>
+      </header>
+      
+      <hr>
+
+      <div class="tagline">
+        <h3>Because two brains <strike>taste</strike> <span>are</span> better than one.</h3>    
+      </div>
+
+      <article>
+        <aside class="top">
+          <h2>Meet your Match!</h2>
+          <p>Have a great idea for a product, but need help making it a reality? We're here to help. Startup Matchmaker is the best place for designers and developers to find each other.</p>
+          <div class="button-top">
+            <h4>Create a profile</h4>
+          </div>
+        </aside>
+      </article>
+
```

Comment:
If you wrapped your `<aside>` inside of a container and gave that same container some padding. Thaht would have given the negative space around those `<aside>`.